client/src/components/related_items/RelatedItemCard.jsx

```diff
@@ -28,12 +30,16 @@ const RelatedItemCard = ({card,  currProduct, handleRelatedItemClick}) => {
         <div>
             <Pre_Modal key={card.id} card={card}  currProduct={currProduct} />
             <div className='RelatedItemCard'>
-                    <div className='RelatedItemCardText' onClick={()=>{handleRelatedItemClick(card.id)}} >
+                    <div onClick={()=>{handleRelatedItemClick(card.id)}} >
                         <img className="image" src={`${photoURL}`} /> 
-                        <p className="category">{card.category}</p>
-                        <h3 className='productName'>{card.name}</h3>
-                        <h4 className='price'>${card.default_price}</h4>
-                        <p>{card.rating}</p>
+                        <div className='cardText'>
+                            <p className="category">{card.category}</p>
+                            <h3 className='productName'>{card.name}</h3>
+                            <h4 className='price'>${card.default_price}</h4>
```

Comment:
I just checked BRD, I think you might need to consider the sale price situation here.